kotest-property/src/desktopMain/kotlin/io/kotest/property/seed/loadSeedFromFile.kt

```diff
@@ -0,0 +1,7 @@
+package io.kotest.property.seed
+
+import io.kotest.common.TestPath
+
+actual fun readSeed(path: TestPath): Long? = null
+actual fun writeSeed(path: TestPath, seed: Long) {}
```

Comment:
warn/error log that saving seeds is not implemented on JS/Desktop?

Reply:
Good idea.

Reply:
Hmm now I think about it, it's not optional per platform. It would just be loads of warnings everytime.